site/docs/v1/tech/lambdas/epic-games-reconcile.adoc

```diff
@@ -0,0 +1,63 @@
+---
+layout: doc
+title: Epic Games Reconcile
+description: An overview of the Epic Games Reconcile lambda
+---
+
+:sectnumlevels: 0
+
+== Epic Games Reconcile lambda
+
+When an Epic Games identity provider is used to complete a federated login request FusionAuth. Please note, there are multiple linking strategies when attempting to reconcile the user. Epic Games defaults to `CreatePendingLink`. You may optionally utilize a lambda to customize the user and user registration during this authentication event.
```

Comment:
> When an Epic Games identity provider is used to complete a federated login request FusionAuth.

This doesn't sound like a complete sentence.

Reply:
> Please note, there are multiple linking strategies when attempting to reconcile the user. Epic Games defaults to `CreatePendingLink`.

This seems like it belongs in a different paragraph or not in this section at all. Does the linking strategy have anything to do with the lambda? The one time it would impact the lambda is if you configure this to be `LinkAnonymously` the lambda will not be used.

Same goes for all of the sections below with this statement.